nemo/collections/asr/data/data_simulation.py

```diff
@@ -0,0 +1,1324 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import shutil
+import warnings
+
+import numpy as np
+import pyroomacoustics as pra
+import soundfile as sf
+import torch
+from gpuRIR import att2t_SabineEstimator, beta_SabineEstimation, simulateRIR, t2n
+from pyroomacoustics.directivities import CardioidFamily, DirectionVector, DirectivityPattern
```

Comment:
`pyroomacoustics` is not part of requirements, can you add it in README to install this as well along with gpuRIR